### PR TITLE
added drag and drop to selected files container

### PR DESCRIPTION
**Pull request by AshishSingh2001**
Fixes: #6 

added drag and drop as mentioned in issue #6  using [desktop_drop](https://pub.dev/packages/desktop_drop) for selected container in Dashboard

### CODE REVIEW

**Comment by Techno-Disaster** [comment]
strange I would have expected some windows and macOS registrants generated too. Have you tested this on windows or macOS? Maybe you don't  have windows and macOS enabled in your flutter config desktop so they weren't generated?

**Comment by Techno-Disaster** [comment]
Tests failed can you look into it?

**Comment by AshishSingh2001** [comment]
i could try checking it in windows and will see why the tests failed

File: lib/bloc_observer.dart [hunk 1]
```diff
@@ -14,21 +14,14 @@ class SimpleBlocObserver extends BlocObserver {
...
-  //   // CurrentSettingsState and ProcessState have a lot of spam
-  //   if ({transition.nextState, transition.nextState}
-  //           .contains(CurrentSettingsState) ||
-  //       {transition.nextState, transition.nextState}.contains(ProcessState)) {
-  //     print(transition);
-  //   }
-  //   super.onTransition(bloc, transition);
-  // }
+  @override
+  void onTransition(Bloc bloc, Transition transition) {
```
Comment (Techno-Disaster) [inline lib/bloc_observer.dart:21]: IIRC this creates a lot of verbose logs during extraction.

**Review by Techno-Disaster — CHANGES_REQUESTED** [review]

File: lib/bloc_observer.dart (hunk 1 above)
Comment (AshishSingh2001) [reply lib/bloc_observer.dart]: reverted the bloc observer changes in other commit

**Comment by Techno-Disaster** [comment]
> i could try checking it in windows and will see why the tests failed

did you test it on windows?

**Comment by AshishSingh2001** [comment]
The drag and drop is working fine in windows
I found some parts of the drag and drop different from the normal file select will make a updated PR soon

**Comment by Techno-Disaster** [comment]
#41

**Pull request closed without merging**